README.md

```diff
@@ -4,35 +4,44 @@ OpenTelemetry aware logger for Cloudflare Workers and Vercel Edge Functions.
 
 ## Usage
 
-```
+```bash
 npm i @baselime/edge-logger
 ```
 
+Add the API key
+
+```bash
+npx wrangler secret put BASELIME_KEY <your Baselime api key>
+```
+
 Add the Logger to your Cloudflare worker
 
-```javascript
+```typescript
+import { BaselimeLogger } from '@baselime/edge-logger'
+
+export interface Env {
+	BASELIME_KEY: string
+}
+
 export default {
-    async fetch(req: Request, env: Env, ctx: ExecutionContext): Promise<Response> {
+	async fetch(req: Request, env: Env, ctx: ExecutionContext): Promise<Response> {
 		const logger = new BaselimeLogger({
 			ctx,
 			apiKey: env.BASELIME_KEY,
 			service: 'my-worker',
 			dataset: 'cloudflare',
 			namespace: 'fetch',
-			requestId: req.headers.get('cf-ray'),
+			requestId: crypto.randomUUID(),
 		})
-		
+
 		logger.info('Hello world', { cfRay: req.headers.get('cf-ray'), foo: 'bar' })
-		// To send a message on a queue, we need to create the queue first
-		// https://developers.cloudflare.com/queues/get-started/#3-create-a-queue
-		await env.MY_QUEUE.send({
```

Comment:
Removed this as it was confusing out of context. Added a link to the example below instead